clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -529,4 +536,7 @@ public enum ConsumerAssignmentStrategy {
     }
   }
 
+  private String getConsumerThreadName(int index) {
```

Comment:
This looks fragile... seems like we're doing string concatenation following some sort of convention, where the `DaemonThreadFactory` will append  "-t" + a number, and in this function here we're replicating the same logic... but nothing guarantees that these two logics are going to remain the same, neither in terms of code structure (preferred), nor even with a unit test...

Reply:
sure, will do